alibi/explainers/integrated_gradients.py

```diff
@@ -477,7 +593,7 @@ def build_explanation(self,
         data.update(predictions=predictions)
 
         # calculate convergence deltas
-        deltas = _compute_convergence_delta(self.model, attributions, baselines, X, target)
-        data.update(deltas=deltas)
+        delta = _compute_convergence_delta(self.model, self.input_dtypes, attributions, baselines, X, target)
+        data.update(deltas=delta)
```

Comment:
It's not a big deal but maybe the history of the changed name could be interesting :) : 
I think originally it was deltas because there's one delta for each data point in the batch. 
For multiple inputs, at the beginning I thought there should be also one delta for each input, so I changed the old deltas into delta to free deltas and assign it to the final list.
Then I realized that even with multiple inputs there's only one delta for each input, so I got rid of a loop over the inputs.
In this process deltas became delta. I changed it back to deltas